src/scss/_main.scss

```diff
@@ -188,8 +188,10 @@ nav {
     margin: 0;
     padding: 0;
 
+    color: var(--color-black);
+
```

Comment:
i'd prefer to stay away from using pure black here. is it needed to pass checks still?